libs/tokens/src/utils/tokenMapToListWithLogo.ts

```diff
@@ -7,6 +7,6 @@ import { TokensMap } from '../types'
  */
 export function tokenMapToListWithLogo(tokenMap: TokensMap): TokenWithLogo[] {
   return Object.values(tokenMap)
-    .sort((a, b) => (a.symbol > b.symbol ? 1 : -1))
+    .sort((a, b) => a.symbol.localeCompare(b.symbol))
```

Comment:
funny, i just came to suggest this, and I saw it was done moments ago!

Reply:
Got to it when reading https://levelup.gitconnected.com/cross-browser-crazy-44e90d61b204